packages/flutter/test/widgets/drawer_test.dart

```diff
@@ -327,7 +327,13 @@ void main() {
     scaffoldKey.currentState.openDrawer();
     await tester.pump(const Duration(milliseconds: 100));
 
-    expect(semantics, includesNodeWith(actions: <SemanticsAction>[SemanticsAction.tap]));
+    expect(
```

Comment:
Split this into 2 statements like below.

Reply:
Done.

Reply:
Will do, will make it more readable and understandable that I am just adding the Dismiss check as well.